### ISSUE TITLE
Missing keyboard navigation (AKA tabbing)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#50]
```
In the main window you cannot move between the controls using Tab /
Shift+Tab keys. The same goes for the settings tab. This behavior was
present in original PG2 too.

Using 0.9.2 (r86) on XPSP3 32 bit.
```

Original issue reported on code.google.com by `Marvello...@gmail.com` on 8 Aug 2009 at 7:40

**Comment by GoogleCodeExporter** [comment]
```
Will get this fixed.  Can't make any promises, but I'd like to get this in 
there by 
1.0.  Might slip a bit though, as I wouldn't want to hold up our official 
Signed 
Driver Release for this.
```

Original comment by `peerbloc...@gmail.com` on 10 Aug 2009 at 8:51
- Changed state: **Accepted**
- Added labels: **Milestone-Release1.0**, **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Not going to make it into 1.0.
```

Original comment by `peerbloc...@gmail.com` on 24 Sep 2009 at 1:08
- Removed labels: **Milestone-Release1.0**
